ConwaysGameOfLife.Tests/InputParserShould.cs

```diff
@@ -6,65 +6,69 @@ namespace ConwaysGameOfLife.Tests
 {
     public class InputParserShould
     {
+        InputParser _parser;
+        public InputParserShould()
+        {
+            _parser = new InputParser();
+        }
         [Theory]
         [InlineData("0,0", 0, 0)]
         [InlineData("-1,-1", -1, -1)]
         [InlineData("100,8", 100, 8)]
-        public void WhenUserEntersStringOfOneAliveCellLocation_ReturnsListOfLocationContainingOnlyOneValue(string userInput, int x, int y) //TODO: shorten method name
+        [InlineData("a,0", -1, 0)]
+        [InlineData("p,s", -1, -1)]
+        [InlineData("3,p", 3, -1)]
+        [InlineData("*,2", -1, 2)]

```

Comment:
I'm not sure how to approach this @indianbread any suggestions? Potentially a `TryParse` function which returns true/false for whether the parse was successful and a Location output into a variable when it is successful.